Project/Src/Document/LineManager/EndOfLineType.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ICSharpCode.TextEditor.Document
+{
+    public enum EndOfLineType
```

Comment:
Maybe?
```suggestion
    public enum EndOfLineMarker
```
...or to be consistent with existing lingo:
```suggestion
    public enum EolMarker
```

Reply:
@RussKie done, thanks!